project-base/webpack.config.js

```diff
@@ -62,8 +63,7 @@ Encore
     }))
     .addPlugin(new CopyPlugin([
         { from: 'web/bundles/fpjsformvalidator', to: '../../assets/js/bundles/fpjsformvalidator', force: true },
-        { from: 'assets/public', to: '../../web/public', force: true },
-        { from: 'node_modules/@shopsys/framework/public/svg/admin', to: '../../web/public/admin/svg', force: true }
```

Comment:
are you sure about this? It seems I've lost svg icons in admin after I removing this line

Reply:
Yes, svg files from admin are loaded in `generateWebFont` (renewal feature, because we removed it in first implementation). It was my mistake, I thought that files are copied automaticaly.

Reply:
ah, ok, thanks for the answer, I will try to reproduce your steps from this PR in my project, thanks for the quick reaction :+1: